Gems/Compression/Code/Source/CompressionSystemComponent.h

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Component/Component.h>
+#include <AzCore/Component/TickBus.h>
+#include <Compression/CompressionBus.h>
+
+namespace Compression
+{
+    class CompressionSystemComponent
+        : public AZ::Component
+        , protected CompressionRequestBus::Handler
+        , public AZ::TickBus::Handler
```

Comment:
This probably came from the template, but maybe the `TickBus` can be removed from this system component.

Reply:
Yeah it is not needed at this time.
It does come from the Default Gem Template.